tests/util_test/util.hpp

```diff
@@ -124,6 +124,9 @@ inline auto make_node_cfgs(uint count) {
       cfg.chain.final_chain.state.genesis_balances[root_node_addr] = 9007199254740991;
       auto& dpos = *cfg.chain.final_chain.state.dpos;
       dpos.genesis_state[root_node_addr][root_node_addr] = dpos.eligibility_balance_threshold;
+      // As test are badly written let's disable it for now
+      cfg.chain.final_chain.state.execution_options.disable_nonce_check = true;
```

Comment:
This will cause lots of unit tests disable nonce check. Which unit tests will test nonce checking?

Reply:
With this change we have 2 levels of nonce checking, first one when we verifying transaction and second one when we execute transaction. This flag is disabling second one, that is part of EVM and not this PR. Anyway I am not disabling anything new, just moving this flag only for tests. If we want to enable it by default, we need to rework a lot of tests and that's for separate task

Reply:
I see. Thanks!